### DISCUSSION TITLE
Direction for react-query Docs Contributions

### DISCUSSION THREAD

**Discussion by banagale** [TanStack/query#1154] [Ideas] [answered]
I was curious if there was any specific direction requested for general "improvements" to the docs.

For example, I want a table of contents or other hyper-linked index to reference material available on the first docs page. 

The reason for this is that in my learning process I've scrolled and clicked next enough times that I had set bookmarks to get back to the query and mutate pages.

I have considered if the docs present a convincing case for the library prior to usage information on purpose, though.

So before I make "improvement" pull requests to the docs, I wanted to first ask if there was any direction or if any reasonable, well-formatted change are likely to be accepted.

**Comment by tannerlinsley** [comment, marked as answer]
Improvements are always welcome, but I’m not quite sure I understand the improvement you are describing yet.

Tanner Linsley

**Reply by banagale** [reply]
You know, I just looked at it again and I had not noticed the "Menu" dropdown this entire time on the docs page.  I use an XDR and so my browser windows are consistently 1000px wide (1/3rd the display), which meant I also wasn't seeing the ToC sidebar on the left side. 🙄

Many sites are a bit off in responsive elements <= 1000px but in this case I wasn't looking for / seeing the Menu menu. I kept wondering, "why are the docs like this?" but the full width search bar should have been a dead giveaway.

So not seeing either the Menu menu or the sidebar, I was suggesting the opening page more like the [reactjs.org Docs link](https://reactjs.org/docs/getting-started.html) page that is more of an index / abstract of topics.

I see Formik's doc's collapses the same. I might look at whether the main content column can be more forgiving of lower width like the react docs. 

Or perhaps at least I'd consider a copy change for the "Menu" menu to "Index of Topics" or something a bit more descriptive.
